maywood/style-editor.css

```diff
@@ -1359,6 +1359,98 @@ body {
 	font-size: 15px;
 }
 
+.site-header .main-navigation {
+	/**
```

Comment:
not critical - wonky indentation

Reply:
@glendaviesnz Ah, that's the automatically-built CSS though.
I'll leave this as it is, as next time someone will touch the source this will be messed up again anyway. 🙂